### DISCUSSION TITLE
reserve seat and pay at entrance

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3761] [Q&A]
Hello.

We have a case that we would like to create events and products and give the visitors the opportunity to actually pay for the tickets on box office. The online system will work as a booking system to only reserve a seat. The payment will occur at the entrance of a museum.
What is the best approach for this ? Can we use the manual payment core plugin ?

Thank you in advance

**Comment by pc-coholic** [comment]
Yes, you could use the Manual Payments plugin.

Just make sure to set acceptable payment expiration deadlines and/or disable the payment reminders and/or disable the automatic order expiration.

Just keep in mind that having an event set up like that will make it quite easy for trolls to completely book your available inventory without showing up and paying.
